WORKSPACE

```diff
@@ -379,10 +379,7 @@ http_archive(
 http_file(
     name = "qemu-img",
     sha256 = "669250ad47aad5939cf4d1b88036fd95a94845d8e0bbdb05e933f3d2fe262fea",
-    urls = [
-        "https://dl.fedoraproject.org/pub/fedora/linux/releases/31/Everything/x86_64/os/Packages/q/qemu-img-4.1.0-2.fc31.x86_64.rpm",
-        "https://storage.googleapis.com/builddeps/669250ad47aad5939cf4d1b88036fd95a94845d8e0bbdb05e933f3d2fe262fea",
-    ],
+    urls = ["https://storage.googleapis.com/builddeps/669250ad47aad5939cf4d1b88036fd95a94845d8e0bbdb05e933f3d2fe262fea"],
```

Comment:
@rmohr I see that the only link left is the one pointing to the builddeps cache. Is there an easy way to implement logic to fetch new original source links somehow that you can imagine?